indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/SeekableStreamIndexTaskRunner.java

```diff
@@ -1973,7 +1963,7 @@ public void resume() throws InterruptedException
   @Produces(MediaType.APPLICATION_JSON)
   public DateTime getStartTime(@Context final HttpServletRequest req)
   {
-    authorizationCheck(req, Action.WRITE);
```

Comment:
I noticed that the refactor changes the Action type to be based on method type, with GET/HEAD being READ and others write. This resource seems to be an exception. Is the earlier WRITE requirement a bug?

Reply:
Good eye! 😃 .
Yes, this was a bug. It should have been READ